pkg/config/config_template.yaml

```diff
@@ -147,6 +147,13 @@ api_key:
 # The port for the go_expvar server
 # dogstatsd_stats_port: 5000
 #
```

Comment:
Good job noticing this was missing from the template 👍

Reply:
Thanks :)